structuralcodes/codes/ec2_2023/_section_8_1_bending.py

```diff
@@ -0,0 +1,601 @@
+from typing import Literal
+
+import numpy as np
+from numpy.typing import ArrayLike
+
+
+def MEd_min(h: float, NEd: float) -> float:
+    """Minimum eccentricity for effects of imperfections.
+
+    EN1992-1-1:2023 Eq.(8.1).
+
+    Computes the minimum moment for a determined h-height section for taking
+    into consideration geometric imperfections unless second order effects are
+    used.
+
+    Args:
+        h (float): Height of the element in mm.
+        Ned (float): Axial force in kN.
+
+    Returns:
+        float: minimum design moment in kNm.
+    """
+    ed_min = max(h / 30, 20) / 1000
```

Comment:
Here can we use the new function e_min? 

```suggestion
    ed_min = e_min(h)
```